Makefile

```diff
@@ -26,10 +26,6 @@ export BIN_DIR=$(PROJ_ROOT)/bin
 export GTEST_DIR=$(PROJ_ROOT)/lib/googletest/googletest
 export PARSER_DIR=$(PROJ_ROOT)/lib/nginx-configparser
 
-.PHONY: deploy
-deploy: compile
-	cd deploy && bin/webserver base_config
-
```

Comment:
The only issue is that since we're not bundling our IP/key with the repo, I'm not sure how to launch the script (since it takes those as arguments)